packages/quick_actions/quick_actions_android/android/src/main/java/io/flutter/plugins/quickactions/QuickActionsPlugin.java

```diff
@@ -96,12 +96,20 @@ public boolean onNewIntent(@NonNull Intent intent) {
     if (intent.hasExtra(QuickActions.EXTRA_ACTION) && activity != null) {
       Context context = activity.getApplicationContext();
       String shortcutId = intent.getStringExtra(QuickActions.EXTRA_ACTION);
-      quickActionsFlutterApi.launchAction(
-          shortcutId,
-          value -> {
-            // noop
-          });
-      ShortcutManagerCompat.reportShortcutUsed(context, shortcutId);
+      if (shortcutId != null) {
```

Comment:
This check was added because there was a warning about `shortcutId` being potentially null in the call to `launchAction`. Probably this never actually happens, but if it does, better to do nothing than to crash.